samples/bluetooth/peripheral_ht/README.rst

```diff
@@ -0,0 +1,26 @@
+.. _peripheral_ht:
+
+Bluetooth: Peripheral HT
+########################
+
+Overview
+********
+
+Similar to the :ref:`Peripheral <ble_peripheral>` sample, except that this
```

Comment:
I'd prefer this to be a complete sentence, but I notice many of the Bluetooth samples are written with a similar sentence fragment, so I won't pick on yours.